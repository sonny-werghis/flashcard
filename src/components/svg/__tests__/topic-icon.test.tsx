import React from 'react';
import { render } from '@testing-library/react-native';
import { TopicIcon } from '../topic-icon';
import { WHITE } from '@constants/colors';

describe('TopicIcon', () => {
  it('should match snapshot', () => {
    const tree = render(<TopicIcon focused={true} color={WHITE} />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
