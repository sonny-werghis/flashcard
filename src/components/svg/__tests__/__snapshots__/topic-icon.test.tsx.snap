// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopicIcon should match snapshot 1`] = `
<RNSVGSvgView
  align="xMidYMid"
  bbHeight={28}
  bbWidth={28}
  fill="#fff"
  focusable={false}
  height={28}
  meetOrSlice={0}
  minX={0}
  minY={0}
  style={
    Array [
      Object {
        "backgroundColor": "transparent",
        "borderWidth": 0,
      },
      Object {
        "flex": 0,
        "height": 28,
        "width": 28,
      },
    ]
  }
  vbHeight={24}
  vbWidth={24}
  width={28}
>
  <RNSVGGroup
    fill={4294967295}
    propList={
      Array [
        "fill",
      ]
    }
  >
    <RNSVGPath
      d="M0 0h24v24H0z"
      fill={null}
      propList={
        Array [
          "fill",
        ]
      }
    />
    <RNSVGPath
      d="M4 6H2v14c0 1.1.9 2 2 2h14v-2H4V6zm16-4H8c-1.1 0-2 .9-2 2v12c0 1.1.9 2 2 2h12c1.1 0 2-.9 2-2V4c0-1.1-.9-2-2-2zm-1 9H9V9h10v2zm-4 4H9v-2h6v2zm4-8H9V5h10v2z"
    />
  </RNSVGGroup>
</RNSVGSvgView>
`;
